test/apiEndpoint.test.js

```diff
@@ -7,22 +7,19 @@ require('babel-polyfill'); // this is needed to make async/await work
 
 // this just tests that the code for responding to a get request is right
 test('api should respond with the right data', async () => {
+  jest.setTimeout(90000);
```

Comment:
I think you can put 90000 as the third argument when calling test if you wanted to make the code more concise. Not that your way isn't bad - makes it abundantly clear what's happening with the code.

Reply:
oh I didn't know that, thanks for the tip